TUT/STA130F24_TUT06_Oct25.ipynb

```diff
@@ -0,0 +1,12137 @@
+          15811626,
+          20654738,
+          24334031,
+          28089915,
+          15774093,
+          20836881,
+          20739849,
+          22133464,
+          22321138,
+          22321138,
+          20704716,
+          20911153,
+          22656843,
+          23445874,
+          14896892,
+          12343580,
+          22037020,
+          23884809,
+          23029213,
+          21841232,
+          21337518,
+          16009583,
+          28086477,
+          19345315,
+          18585146,
+          25244974,
+          23877257,
+          22394733,
+          20614959,
+          20083786,
+          15048037,
+          28531069,
+          23143560,
+          24616128,
+          20604712,
+          24614028,
+          27539205,
+          38783854,
+          16714699,
+          17841891,
+          29244150,
+          28348062,
+          25117617,
+          25634843,
+          27038225,
+          22786507,
+          22786507,
+          22905607,
+          25993467,
+          27591483,
+          30085953,
+          40993951,
+          18766305,
+          22494434,
+          18260267,
+          27722511,
+          26350323,
+          21736748,
+          21392376,
+          21730051,
+          26640595,
+          26640595,
+          28476232,
+          26940627,
+          33851356,
+          23951914.29,
+          23951914.29,
+          31485172.45,
+          26894099.17,
+          28842985.58,
+          28706805.95,
+          28477729.82,
+          28923675.96,
+          27125138.45,
+          25496118,
+          25496118,
+          24062535.05,
+          24062535.05,
+          23009994.04,
+          21495765.02,
+          25903635.94,
+          35280351.26,
+          29567891.31,
+          25872048.71,
+          21686611.83,
+          21686611.83,
+          21078311.83,
+          23822896.59,
+          27675629.09,
+          27675629.09,
+          34447048.43,
+          35186711.31,
+          41151034.39,
+          32862919.91,
+          33603536.36,
+          32793984.32,
+          29082683.33,
+          29082683.33,
+          27899472.03,
+          26613186.99,
+          30239538.42,
+          28628692.39,
+          31147315.59,
+          27685184.24,
+          27685184.24,
+          27685184.24,
+          31954759.69,
+          42596195.27,
+          37729078.44,
+          36994026.79,
+          36994026.79,
+          38813251.1,
+          39431767.13,
+          39653551.79,
+          40285883.82,
+          37330027.69,
+          37230401.33,
+          36501888.28,
+          39865469.32,
+          41012369.65,
+          37530273.1,
+          27159271.45,
+          27159271.45,
+          34204242.15,
+          35291615.88,
+          34725461.34,
+          35439598.96,
+          34493610.47,
+          35987055.01,
+          29894777.05,
+          29894777.05,
+          30098714.42,
+          30098714
```

Comment:
Hmm.. yes, this is what I was afraid of... this is more of a demo for MLR not SLR... this is why coordination with Serena is needed.

Reply:
Anyway, I'll be taking over the indicator variable presentation and example demo as part of my Nov04 LEC 07, and you'll be reimagining what this example should be for the Oct 25 TUT 06 demo.

Reply:
It's supposed to be a comment on lines 12078-12083:

"### Step 1: Students should come up with the standard form of the regression equation for $E[\\text{avg_ticket_price}] $ for both graphs\n",
     "### Step 2: Then, they should collectively try to find estimates for the parameters of the equations.\n",
     "\n",
     "- Re-emphasize here that in Step 1, they were talking about the idea of a linear model behind the two variables of the broadway dataset and in Step 2 they manually fit a linear model to the indicator and outcome variable.\n",
     "<br><br>\n",
     "- Discuss students' answers and confirm estimates of parameters by scrolling over the trendlines."